### PR TITLE
dark-light mode

### PR DESCRIPTION
**Pull request by culinovic-a**

### CODE REVIEW

File: src/app/core/layout/layout.component.html [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+  <mat-toolbar color="primary">
+    <div class="nav">
+      <button mat-icon-button (click)="snav.toggle()">
+        <mat-icon>menu</mat-icon>
+      </button>
+      <h1>Notes</h1>
+    </div>
+    <mat-slide-toggle
+      (click)="changeTheme()"
+      matTooltip="Change Theme"
```
Comment (tabuckner) [inline src/app/core/layout/layout.component.html:11]: nice usage of tooltips!!

File: src/app/core/layout/layout.component.ts [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 })
 export class LayoutComponent implements OnInit {
   constructor() {}
 
+  isLightTheme = false;
+
   ngOnInit() {}
+
+  changeTheme() {
+    if (this.isLightTheme) {
```
Comment (tabuckner) [inline src/app/core/layout/layout.component.ts:16]: this is fine, but you can even shorten this to:
```suggestion
  this.isLightTheme = !this.isLightTheme
```

That's a nice shorthand if you're JUST toggling a boolean

File: src/app/light-theme.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import '~@angular/material/theming';
+@include mat-core();
+.light-theme {
```
Comment (tabuckner) [inline src/app/light-theme.scss:3]: nice :) 

On my most recent project at work I'm applying this sort of class to my top level component (could be a body or even the HTML tag if you wanted). 

Great minds think alike :)

File: src/app/core/layout/layout.component.ts (hunk 2 above)
Comment (culinovic-a) [reply src/app/core/layout/layout.component.ts:16]: > this is fine, but you can even shorten this to:
> 
> That's a nice shorthand if you're JUST toggling a boolean

I actually wrote it like this first, but in my last commit changed - I have something in my mind for later, more about that on Slack
